index.js

```diff
@@ -218,6 +220,7 @@ Lexer.prototype = {
       var tok = this.tok('comment', captures[2]);
       tok.buffer = '-' != captures[1];
       this.pipeless = true;
+      this.interpolationAllowed = tok.buffer;
```

Comment:
Should we be allowing interpolation inside comments?  I think a lot of people expect comments to be fully inert even when they're buffered.